packages/cli/src/commands/dbCommands/studio.js

```diff
@@ -1,32 +1,26 @@
 import path from 'path'
 import fs from 'fs'
 
-import terminalLink from 'terminal-link'
-
 import { runCommandTask, getPaths } from 'src/lib'
+import { schema, epilogue } from 'src/commands/dbCommands/options'
 import c from 'src/lib/colors'
 
 export const command = 'studio'
 export const description = 'Start Prisma Studio'
 
 export const builder = (yargs) => {
-  yargs.epilogue(
-    `Also see the ${terminalLink(
-      'Redwood CLI Reference',
-      'https://redwoodjs.com/reference/command-line-interface#studio'
-    )}`
-  )
+  yargs.option('schema', schema()).epilogue(epilogue())
 }
 
-export const handler = async () => {
+export const handler = async ({ schema }) => {
   // No schema, no studio.
-  if (!fs.existsSync(getPaths().api.dbSchema)) {
+  if (!fs.existsSync(schema())) {
```

Comment:
```suggestion
  if (!fs.existsSync(schema)) {
```
I would assume that `schema` is a string at this point?

Reply:
Correct, fixed in https://github.com/redwoodjs/redwood/pull/1424/commits/1a82c434c7c025a6a8a3f7d387214e0c5efc89bb.